torch/csrc/jit/serialization/export.cpp

```diff
@@ -1366,10 +1366,22 @@ void check_onnx_proto(const std::string& proto_string, bool full_check) {
     throw std::runtime_error("Invalid ONNX proto string.");
     return;
   }
+  // baseline check
   onnx::checker::check_model(model);
-
+  onnx::shape_inference::InferShapes(model);
+  // full check including strict shape inference check
+  // Notice that onnx::checker::check_model doesn't
```

Comment:
```suggestion
  // NOTE: onnx::checker::check_model doesn't
```

Reply:
I would reword this to make the intention clear

Reply:
Done